### PR TITLE
success criterias for for Technology Extractor widget

### PR DESCRIPTION
**Pull request by mshchygol**

### CODE REVIEW

File: tests/testtechnologyextractor.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /**
  * @fileoverview Defines tests for Technology Extractor widget.
+ * Success criterias:
+ * - element with id 'content-technology-container' should exist;
```
Comment (vpodk) [inline tests/testtechnologyextractor.js]: https://en.wikipedia.org/wiki/Sentence_(linguistics)

Comment (mshchygol) [inline tests/testtechnologyextractor.js]: Fixed

**Comment by mshchygol** [comment]
LGTM 👍

File: tests/testtechnologyextractor.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 /**
  * @fileoverview Defines tests for Technology Extractor widget.
+ * Success criterias:
+ * - Element with id 'content-technology-container' should exist;
+ * - Result table with id 'technology-data-table' or element with class
+ *  'rule true' should exist;
```
Comment (zalesky) [inline tests/testtechnologyextractor.js:6]: add space

Comment (zalesky) [inline tests/testtechnologyextractor.js:6]: It will be great if we'll be using the same quotes ' or " or `

**Pull request closed without merging**